README.md

```diff
@@ -5185,14 +5221,27 @@ cd /opt/nemo-rlhf \
 && python -u examples/nlp/gpt/train_gpt_ppo_actor.py \
     --config-path=examples/nlp/gpt/conf/ \
     --config-name=gpt_ppo_actor \
-    "model.data.data_prefix={train: [/path/to/train_data], validation: [/path/to/val_data], test: [/path/to/test_data]}" \
-    model.pretrained_checkpoint.restore_from_path=/path/to/model.nemo
+    exp_manager.explicit_log_dir=/path/to/actor_output_dir \
+    "model.data.data_prefix={train: [/path/to/actor_train], validation: [/path/to/actor_val], test: [/path/to/actor_test]}" \
+    model.pretrained_checkpoint.restore_from_path=/path/to/sft_model.nemo
 ```
-This command will launch the PPO Actor job on the local computer. All the configuration parameters can be modified in the `gpt_ppo_actor.yaml` file, or by overriding them through the CLI command.
 
-##### 5.16.2.5 Launching every job at once with SLURM
-<a id="markdown-launching-every-job-at-once-with-slurm" name="launching-every-job-at-once-with-slurm"></a>
-Heterogeneous jobs can be used to launch all four jobs simultaneously in different nodes, using a script like the one shown next:
+This command will launch the PPO Actor job on the local computer. All the configuration parameters can be modified in the `gpt_ppo_actor.yaml` file, or by overriding them through the CLI command: in particular, the Actor's model config should match the one used to train the SFT model, and you may need to provide the correct name of the tokenizer files as described in the RM training section above.
+
+The data files should point to the names of datasets (without the ".bin" or ".idx" suffix) generated in a manner similar to what is described in the RM training section, but with an important difference: they should only contain prompts.
```

Comment:
This is my guess, but I haven't actually found corresponding .jsonl files on our cluster ==> please confirm.

Reply:
that's correct, the data processing script needs to generate one with just the prompts and one with prompts + responses. I will update the doc with that